sigmadb.py

```diff
@@ -91,7 +91,7 @@ def main():
 
                 action = ""
                 while action != "1":
-                    action = input_utils.get_input_matching("1 - exit\n2 - browse movies\n3 - manage followed users\n4 - create collection\n5 - browse collections\n", regex='[123456]')
+                    action = input_utils.get_input_matching("1 - exit\n2 - browse movies\n3 - manage followed users\n4 - create collection\n5 - browse collections\n6 - see what's popular\n>", regex='[123456]')
```

Comment:
NIT:
It might be more appropriate to label the submenu "Recommended Movies" since "For You" is not necessarily popular.